### ISSUE TITLE
[Community Action Required] Consider watching new module repos

### ISSUE THREAD

**Issue by ddorwin** [immersive-web/webxr#816]
tl;dr: If you are watching this repo and received an email for this issue, you may also want to subscribe to the new module repos.

* https://github.com/immersive-web/webxr-gamepads-module/ (part of "VR Complete" and on the same timeline as the core spec)
* https://github.com/immersive-web/webxr-ar-module/

<br>Details: There are currently 252 people watching this repo (`webxr`). When WebXR was split into modules (#758), none of those subscriptions carried over. There are currently only 5 and 9 people, respectively, watching the repos above, meaning most members of the community are not receiving updates about the work (issues, PRs, and comments) being done in those repos. This is especially important for the first repo, which is really part of the initial WebXR "VR Complete" release.

/agenda Remind people in the agenda and call.

**Comment by toji** [comment]
Very important PSA! Thanks David

**Comment by AdaRoseCannon** [comment]
This was announced via email & in the call I will close this.
